jekyll/_cci2/reusing-config.md

```diff
@@ -8,7 +8,7 @@ order: 1
 ---
 
 
-This document describes how to version your [.circleci/config.yml]({{ site.baseurl }}/2.0/configuration-reference/) file and get started with reusable commands and executors.
+This document describes how to version your [.circleci/config.yml]({{ site.baseurl }}/2.0/configuration-reference/) file and get started with reusable orbs, commands, and executors.
```

Comment:
We should probably add jobs here -- new in 2.1 was the ability to call the same job twice in a workflow, which is particularly handy when using parameters in jobs.

Reply:
added